### PR TITLE
🌐🇪🇦 more translations

### PR DESCRIPTION
**Pull request by PoloSpain**

### CODE REVIEW

File: src/modules/extendedCallWindow/i18n/es_ES.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            pushRight: 'muestra este cuadro encima de la lista de vehículos',
+            pushLeft: 'muestra este cuadro en la posición original',
+        },
+        selected: 'Seleccionado',
+        vehiclesByRequirement: {
+            [/^Camion(es)? de bomberos$/]: [0, 1, 13, 18, 30, 33, 48],
+            [/^Camión(es)? AE$/]: [2, 13],
+            [/^Battalion chief unit(s)?$/]: [3, 12],
+            [/^Vehículo(s)? de rescate(s)?$/]: [4, 8, 18, 21, 28],
+            [/^Vehículo(s) aéreo(s)?$/]: [6],
```
Comment (JRH-1997) [inline src/modules/extendedCallWindow/i18n/es_ES.js:69]: Missing ? after Vehículo(s)

File: src/modules/extendedCallWindow/i18n/es_ES.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            dragging: 'arrastrar ventana',
+            textMode: 'modo de texto on/off',
+            minified: 'minimizar',
+            overlay: 'cubrir',
+            reload: 'recargar',
+            pushRight: 'muestra este cuadro encima de la lista de vehículos',
+            pushLeft: 'muestra este cuadro en la posición original',
+        },
+        selected: 'Seleccionado',
+        vehiclesByRequirement: {
```
Comment (JRH-1997) [inline src/modules/extendedCallWindow/i18n/es_ES.js:64]: You could remove the not needed names.

**Review by JRH-1997 — CHANGES_REQUESTED** [review]

File: src/modules/missionHelper/i18n/es_ES.root.json [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+		},
+		"multifunctionals": {
+			"battalion_chief_vehicles": {
+				"description": "Resta el número de Mando móvil requeridos del UMC y oculta el UMC si el número cae por debajo de 0.",
+				"title": "UMC tasa simple"
+			},
+			"heavy_rescue_vehicles": {
+				"description": "Resta el número de camiones de bomberos necesarios del FUV y oculta los camiones de bomberos si el número cae por debajo de 0. Esta función está destinada a personas que conducen un sistema de vehículos de rescate puro.",
+				"title": "Tasa simple de FUV"
+			},
```
Comment (jxn-30) [inline src/modules/missionHelper/i18n/es_ES.root.json:73]: Add a line with `}` above please.

**Review by jxn-30 — CHANGES_REQUESTED** [review]

File: src/modules/notificationAlert/i18n/es_ES.root.json [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+		"allianceChat": "Mensaje de chat",
+		"allianceChatMention": "Mensaje de chat (sólo menciones)",
+		"allianceChatWhisper": "Mensaje de chat (sólo mensajes privados)",
+		"allianceForum": "Nuevo artículo en el foro de la alianza",
+		"allianceMessage": "Nuevo mensaje de alianza",
+		"allianceNews": "Nuevas noticias de la alianza",
+		"dm": "Nuevo mensaje en la bandeja de entrada",
+		"ingame_news": "Noticias en el juego",
+		"mission_getred": "Una misión requiere atención",
+		"mission_getred_alliance": "Una misión de alianza requiere atención
```
Comment (jxn-30) [inline src/modules/notificationAlert/i18n/es_ES.root.json:20]: And here is missing a `,` at the end of the line.

**Review by jxn-30 — CHANGES_REQUESTED** [review]

File: src/modules/notificationAlert/i18n/es_ES.root.json [hunk 5]
```diff
@@ -17,7 +17,7 @@
 		"dm": "Nuevo mensaje en la bandeja de entrada",
 		"ingame_news": "Noticias en el juego",
 		"mission_getred": "Una misión requiere atención",
-		"mission_getred_alliance": "Una misión de alianza requiere atención
```
Comment (JRH-1997) [inline src/modules/notificationAlert/i18n/es_ES.root.json]: @Suchty112  a "at the end before the ,

Comment (Suchty112) [reply src/modules/notificationAlert/i18n/es_ES.root.json]: yes I change immediately with the last error of jan. thank you

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**